### ISSUE TITLE
doesn't work with v 5.3.0

### ISSUE THREAD

**Issue by thob** [alsvartr/ulauncher-nm#1]
I get ```It looks like this extension is not compatible with the new version of Ulauncher.```

**Comment by mac-linux-free** [comment]
for me too...need this :)

**Comment by rabin-io** [comment]
Just sent a PR #2 , you can try it out , seem to work for me, but a bit slow, maybe the caching not working properly.

**Comment by florianbeer** [comment]
@rabin-io How can I test your branch?

I cloned it into `~/.local/share/ulauncher/extensions` and added a new section to `~/.config/ulauncher/extensions.json` but it only shows up as "stopped" in the extension settings menu.

ulauncher v5.4.0 (Extension API v2.0.0)

**Comment by rabin-io** [comment]
@florianbeer I don't think you need to edit any file, just clone the repo and checkout the python3 branch

    git clone https://github.com/rabin-io/ulauncher-nm.git ~/.local/share/ulauncher/extensions/ulauncher-nm
    cd ~/.local/share/ulauncher/extensions/ulauncher-nm
    git checkout python3
    # restart ulauncher

**Comment by florianbeer** [comment]
Works like a charm. Thank you!

**Comment by luispabon** [comment]
```
2020-02-13 11:23:23,088 | ERROR | ulauncher.ui.windows.PreferencesUlauncherDialog: on_scheme_callback() | GithubExtensionError: Could not find versions.json file using URL "https://raw.githubusercontent.com/alsvartr/ulauncher-nm/master/versions.json"
```

**Comment by rabin-io** [comment]
try using my repo instead

**Comment by kampa133** [comment]
@rabin-io 
https://github.com/rabin-io/ulauncher-nm/tree/python3 does not work
```
2020-06-17 10:20:31,625 | ERROR | ulauncher.ui.windows.PreferencesUlauncherDialog: on_scheme_callback() | GithubExtensionError: Invalid GithubUrl: https://github.com/rabin-io/ulauncher-nm/tree/python3
file:///usr/share/ulauncher/preferences/dist/static/js/app.fed85b0cfd91ce6afc0b.js:4919:18: CONSOLE ERROR TypeError: e.cancel is not a function. (In 'e.cancel()', 'e.cancel' is undefined)
```
If I could I would have opened an issue there.

**Comment by rabin-io** [comment]
The repo is there, and it's working for me right now.

- When do you get this error ?
- Is the plugin working for you ?

![image](https://user-images.githubusercontent.com/5116363/84986315-0b623000-b147-11ea-9cce-1aae86f20e13.png)

**Comment by kampa133** [comment]
I start ulauncher from terminal and the error accures while trying to install the extension. (via pasting: https://github.com/rabin-io/ulauncher-nm/tree/python3) 
![image](https://user-images.githubusercontent.com/14992645/85019426-1fba2300-b16f-11ea-9a00-d1a25ed147fc.png)

The error is "Invalid GithubUrl" so I guess I need to install it on an other way.

It is not working, sorry
(ubuntu 20.04)

**Comment by rabin-io** [comment]
Just download the [python3 branch](https://github.com/rabin-io/ulauncher-nm/releases/download/v1.0/ulauncher-nm-python3.zip) and extract it to the `~/.local/share/ulauncher/extensions` folder.

**Comment by kampa133** [comment]
Great that did the trick!
Thank you very much.

may be you can make it work from the gui to get automatic updates.

**Comment by rabin-io** [comment]
I'm not really maintaining this package, I just helped with the code migration to python3, and shared my work.

**Comment by luispabon** [comment]
I don't believe it works because Ulauncher always loads the master branch. Any chance you could merge your python3 branch into master?

**Comment by rabin-io** [comment]
Done
